entrypoint.sh

```diff
@@ -14,6 +14,11 @@ function check_number() {
 	fi
 }
 
+if [ -z "$APPLICATION_TYPE" ]; then
```

Comment:
Nit: use bash conditional `[[ -z "$APPLICATION_TYPE" ]]`

Reply:
I changed this , regardless, to `[[ -z "foo" ]]` form , but I think I'd like to get to some agreement if we're going to maintain different Bash scripts. I'm fine with anything as long as we agree on something :)

Reply:
Sorry, I actually didn't change it in this particular file, but in the `buid_deploy.sh` script only, as the entrypoint was based on this https://github.com/sclorg/s2i-ruby-container/blob/master/2.7/s2i/bin/run , and is the original entrypoint used in prod.

Reply:
Oh, I now see that this was taken from our prev. s2i. Since we're using `bash`, I'd stick with it's constructs. `[[ ]]` and `==` in this case.